src/pages/zoeken/[slug]/[ie]/index.tsx

```diff
@@ -1319,17 +1320,46 @@ const ObjectDetailPage: NextPage<ObjectDetailPageProps> = ({ title, url }) => {
 		</>
 	);
 
+	// To determine the correct error page or the object detail page, we follow this flow:
+	// fetch object
+	//    - 200: show detail page
+	//    - 404, 403: check the visitor space info
+	//           - 404: not found error page
+	//           - 410: visitor space no longer available error page
+	//           - 200: check visit request info
+	//                   - 200: not found error page
+	//                   - 404: no access error page
+
 	const renderPageContent = () => {
 		if (mediaInfoIsLoading || visitRequestIsLoading || visitorSpaceIsLoading) {
 			return <Loading fullscreen owner="object detail page: render page content" />;
 		}
-		if (isErrorSpaceNotActive) {
-			return <ErrorSpaceNoLongerActive />;
+		if (mediaInfo) {
+			return <div className="p-object-detail">{renderObjectDetail()}</div>;
 		}
-		if (isErrorNotFound) {
-			return <ErrorNotFound />;
+		if (isMediaInfoErrorNoAccess || isMediaInfoErrorNotFound) {
```

Comment:
Add empty lines between if blocks/statements to make it easier to read / to create a more clear code structure.